front/src/modules/ui/dropdown/components/DropdownButton.tsx

```diff
@@ -20,6 +21,7 @@ type OwnProps = {
   dropdownHotkeyScope?: HotkeyScope;
```

Comment:
Make this mandatory so you can replace `dropdownHotkeyScope?.scope ?? 'dropdown-button'` with `dropdownHotkeyScope.scope`

Reply:
@lucasbordeau Done